### PR TITLE
Fixed typo on ha (attempt 2)

### PR DESCRIPTION
**Pull request by Josipmrden**
### Release note

High availability example updated with additional configuration.

### Checklist:

- [X] Check all content with Grammarly
- [X] Perform a self-review of my code
- [X] The build passes locally
- [X] My changes generate no new warnings or errors

### CODE REVIEW

**Review by as51340 — CHANGES_REQUESTED** [review]
Can you set target branch to memgraph-2.22? Because in that branch there are already HA changes, will be easier to see alltogether

File: pages/clustering/high-availability.mdx [hunk 1]
```diff
@@ -601,77 +549,101 @@ services:
     image: "memgraph/memgraph"
     container_name: coord1
     volumes:
-      - ./license.cypher:/tmp/init/license.cypher:ro
-      - ./HA_register.cypher:/tmp/init/HA_register.cypher:ro
+      - ./license.cypherl:/tmp/init/license.cypherl:ro
```
Comment (as51340) [inline pages/clustering/high-availability.mdx:552]: You changed ports but not in registration queries above.

Comment (as51340) [reply pages/clustering/high-availability.mdx:552]: And change it in chapter Add Coordinator Instance

Comment (as51340) [reply pages/clustering/high-availability.mdx:552]: Also change in Data instances chapter exporting ports and in Coordinator instances also exporting ports.

**Review by as51340 — CHANGES_REQUESTED** [review]

**Review by as51340 — APPROVED** [review]

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**